src/vos/vos_dtx.c

```diff
@@ -648,6 +661,15 @@ do_dtx_rec_release(struct umem_instance *umm, struct vos_container *cont,
 		break;
 	}
 
+	if (unlikely(rc == -DER_NONEXIST)) {
+		struct vos_tls	*tls = vos_tls_get(false);
```

Comment:
Better to use vos_pool->vp_sysdb as parameter.

Reply:
Sure, it will be changed in subsequent patch.